packages/sdk/src/protocols/intents.ts

```diff
@@ -328,7 +329,10 @@ export class IntentGateway {
 		evmChainID: string,
 	): Promise<bigint> {
 		const client = this[gasEstimateIn].client
-		const gasPrice = await client.getGasPrice()
+		const etherscanApiKey = this[gasEstimateIn].config.getEtherscanApiKey()
+		const gasPrice = etherscanApiKey
+			? await getGasPriceFromEtherscan(evmChainID, etherscanApiKey)
```

Comment:
If this line fails revert to client.getGasPrice
```suggestion
			? await getGasPriceFromEtherscan(evmChainID, etherscanApiKey).catch(() => client.getGasPrice())
```